ocd_backend/items/goapi_committee.py

```diff
@@ -10,9 +10,8 @@ def get_object_model(self):
             'organization': self.source_definition['key'],
         }
 
-        committee = Organization(self.original_item['id'], **source_defaults)
+        committee = Organization('committee-' + str(self.original_item['id']), **source_defaults)
```

Comment:
I suspect this doesn't adhere to #162. It's fine for now but should be fixed eventually.

Reply:
I added this to avoid collisions with other items that use only an integer as an ID.